index_impl.go

```diff
@@ -632,6 +574,75 @@ func (i *indexImpl) SearchInContext(ctx context.Context, req *SearchRequest) (sr
 	}, nil
 }
 
+func LoadAndHighLightFields(hit *search.DocumentMatch, req *SearchRequest,
```

Comment:
minor naming thing -- I see existing code calls it "Highlighter", instead of "HighLighter" -- english is weird.

Reply:
done!